data_prep/parse_training_data.py

```diff
@@ -166,15 +194,19 @@ def save_json(self, coverage: bool, group: bool, save_dir: str):
     data_filapath = os.path.join(save_dir, data_filename)
     with open(data_filapath, 'w') as file:
       json.dump(data, file, indent=4)
-    print(f'Saved to {data_filapath}.')
+    logging.info('Saved to: %s', data_filapath)
 
 
 class Experiment:
   """The directory of an experiment, containing benchmark result directories."""
 
-  def __init__(self, experiment_dir: str) -> None:
+  def __init__(self, experiment_dir: str, bucket_url: str = '') -> None:
```

Comment:
Can you add some comments to explain what `bucket_url` is expected to contain, vs `experiment_dir` ?

Reply:
Done. Added a comment here, explanations in arg parser, and an assertion after parsing args.